public/javascripts/SVValidate/src/Main.js

```diff
@@ -209,6 +209,486 @@ function Main (param) {
+                                'top': '325px'
+                            }
+                        }
+                    },
+                    {
+                        'isExampleCorrect': false,
+                        'exampleTypeLabel': i18next.t('mission-tutorial.example-type-label-incorrect'),
+                        'exampleTypeIcon': '#smile-negative',
+                        'slideTitle': i18next.t('mission-tutorial.signal.slide-2.title'),
+                        'slideSubtitle': i18next.t('mission-tutorial.label-type-subtitle'),
+                        'slideDescription': i18next.t('mission-tutorial.signal.slide-2.description'),
+                        'imageURL': 'assets/images/tutorials/signal-incorrect-1.png',
+                        'onImageLabel': {
+                            'title': i18next.t('mission-tutorial.on-image-label-title-incorrect'),
+                            'description': i18next.t('mission-tutorial.on-image-label-description-incorrect'),
+                            'position' : {
+                                'left': '377px',
+                                'top': '86px'
+                            }
+                        }
+                    },
+                    {
+                        'isExampleCorrect': false,
+                        'exampleTypeLabel': i18next.t('mission-tutorial.example-type-label-incorrect'),
+                        'exampleTypeIcon': '#smile-negative',
+                        'slideTitle': i18next.t('mission-tutorial.signal.slide-3.title'),
+                        'slideSubtitle': i18next.t('mission-tutorial.label-type-subtitle'),
+                        'slideDescription': i18next.t('mission-tutorial.signal.slide-3.description'),
+                        'imageURL': 'assets/images/tutorials/signal-incorrect-2.png',
+                        'onImageLabel': {
+                            'title': i18next.t('mission-tutorial.on-image-label-title-incorrect'),
+                            'description': i18next.t('mission-tutorial.on-image-label-description-incorrect'),
+                            'position' : {
+                                'left': '390px',
+                                'top': '203px'
+                            }
+                        }
+                    },
+                    {
+                        'isExampleCorrect': false,
+                        'exampleTypeLabel': i18next.t('mission-tutorial.example-type-label-incorrect'),
+                        'exampleTypeIcon': '#smile-negative',
+                        'slideTitle': i18next.t('mission-tutorial.signal.slide-4.title'),
+                        'slideSubtitle': i18next.t('mission-tutorial.label-type-subtitle'),
+                        'slideDescription': i18next.t('mission-tutorial.signal.slide-4.description'),
+                        'imageURL': 'assets/images/tutorials/signal-incorrect-3.png',
+                        'onImageLabel': {
```

Comment:
I feel like `imageOnLabel` might be quicker to understand than `onImageLabel`. Thoughts?

Reply:
'labelOnImage' would be semantically correct since the variable should refer to the 'label'.
And for the same reason I think it may be better to keep 'label' in the end. But I can change it to 'labelOnImage' if you think that's better.

Reply:
Hahaha oops! Yep that was a typo from me. I did mean to say `labelOnImage` instead of `imageOnLabel` :joy: let's go with `labelOnImage`!

Reply:
haha cool 😆